probe/appclient/app_client_internal_test.go

```diff
@@ -16,6 +15,7 @@ import (
 	"github.com/weaveworks/common/test"
 	"github.com/weaveworks/scope/common/xfer"
 	"github.com/weaveworks/scope/report"
+	"github.com/weaveworks/scope/test/reflect"
```

Comment:
Did this `"reflect"` -> `"github.com/weaveworks/scope/test/reflect"` happen by mistake?